src/app/pipes/remove-string-value.pipe.ts

```diff
@@ -0,0 +1,14 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'removeStringValue'
+})
+export class RemoveStringValuePipe implements PipeTransform {
+
+    transform(value: string, toRemove: string) {
+        if (!toRemove || toRemove === '') {
+            return value;
+        }
+        return value.replace(toRemove, '');
```

Comment:
Not sure if this will ever be an issue, but if the first word is removed it will leave a space at the start.

Reply:
Good catch.  I can trim the result before returning.